test/e2e/suites/e2e/e2e_test.go

```diff
@@ -208,8 +209,13 @@ var _ = Describe("e2e tests", func() {
 
 			shared.Byf("Creating MachineDeployment with custom port options")
 			md3Name := clusterName + "-md-3"
+
+			tTrue := true
 			customPortOptions := &[]infrav1.PortOpts{
-				{Description: "primary"},
+				{
+					Description: "primary",
+					Trunk:       &tTrue,
```

Comment:
nit: you could use `pointer.Bool(true)` here.

Reply:
Thanks, will do that.